apps/public-web/src/EShopOnAbp.PublicWeb/ServiceProviders/PaymentTypeProvider.cs

```diff
@@ -9,15 +9,15 @@ public List<PaymentType> GetPaymentTypes()
     {
         return new List<PaymentType>
         {
-            new() {Id = 0, Name = "Demo", IconCss = "fa-credit-card demo", IsDefault = true},
-            new() {Id = 1, Name = "Paypal", IconCss = "fa-cc-paypal paypal"}
+            new() { Type = "demo", Name = "Demo", IconCss = "fa-credit-card demo", IsDefault = true},
+            new() { Type = "paypal", Name = "Paypal", IconCss = "fa-cc-paypal paypal"}
         };
     }
 }
 
 public class PaymentType
 {
-    public int Id { get; set; }
+    public string Type { get; set; }
```

Comment:
`Name` can be rename to `Type`. Or we can use localized string for `Name`. However I favor about keeping id.